webui/react/src/hooks/useModal/Experiment/useModalExperimentMove.tsx

```diff
@@ -193,7 +180,8 @@ const useModalExperimentMove = ({ onClose }: Props): ModalHooks => {
             style={{ width: '100%' }}
             value={destSettings.workspaceId}
             onSelect={handleWorkspaceSelect}>
-            {workspaces.map((workspace) => {
+            {Loadable.getOrElse([], workspaces).map((workspace) => {
```

Comment:
I didn't see a loading state available for SelectFilter but that should probably be added to that component. Also we really should be passing options as a prop instead of children since that generally improves control and performance.